public_html/wp-content/mu-plugins/wcorg-subroles.php

```diff
@@ -178,3 +181,31 @@ function map_subrole_caps( $primitive_caps, $meta_cap, $user_id, $args ) {
 }
 
 add_filter( 'map_meta_cap', __NAMESPACE__ . '\map_subrole_caps', 10, 4 );
+
+/**
+ * Remove capabilities that are "additional" i.e. stored in user meta.
```

Comment:
```suggestion
* Ignore capabilities that are "additional" i.e. stored in user meta.
```

`MIGHT` - "Remove" could be confused for deleting from the database.

Reply:
Good call. 23c31bd6

Reply:
Oops, should have just used GitHub's UI to commit your suggestion :/